etl/steps/data/garden/war/2023-07-07/mars.meta.yml

```diff
@@ -0,0 +1,66 @@
+dataset:
+  title: History of war (Project Mars, 2020)
+  description: >-
+    This dataset contains information on armed conflicts, both internal and external, in the period of 1800 and 2011.
+
+
+    It has been constructed using the Project Mars dataset by Lyall, Jason.
+
+
+    More details about Project Mars:
+
```

Comment:
Why is this built as a text here + the text from the snapshot? Wouldn't it be better to keep text to modify only in one place?

Reply:
I am still determining why, but the snapshot metadata from Project Mars is appended to the description (?)

Therefore I added this final line before the text from snapshot.

Is this something happening to you too?

Reply:
It's weird, uh? It also happens to me in this case. For my newest datasets, I keep the snapshot descriptions whenever possible, because currently things are carried well. And in the past the text was replaced so this hasn't happened to me before